docs/source/integration.rst

```diff
@@ -152,7 +154,13 @@ See also configuration options for Thoth's client present in `Thamos
 documentation <https://thoth-station.ninja/docs/developers/thamos/>`_.
 
 An example of such application can be found on `GitHub  - s2i TensorFlow
-example <https://github.com/thoth-station/s2i-example-tensorflow>`_.
+example <https://github.com/thoth-station/s2i-example>`_.
+
+.. raw:: html
```

Comment:
https://github.com/fridex/adviser/blob/docs-link/docs/source/integration.rst
The video doesn't show up on GitHub, do we need to worry about it?
I think GitHub doesn't support iframes.